lf.1

```diff
@@ -1388,6 +1388,12 @@ Newer terminals (e.g. gnome-terminal) may prefix the key with an escape key when
     map <a-a> down
 .EE
 .PP
+It is possible to combine special keys with modifiers:
+.PP
+.EX
+    map <a-enter> down
+.EE
+.PP
 Please note that, some key combinations are not possible due to the way terminals work (e.g. control and h combination sends a backspace key instead). The easiest way to find the name of a key combination is to press the key while lf is running and read the name of the key from the unknown mapping error.
```

Comment:
I would reword the first sentence as:

> WARNING: Some key combinations will likely be intercepted by your OS, window manager, or terminal. Other key combinations cannot be recognized by lf due to the way terminals work (e.g. control and h combination sends a backspace key instead). The easiest way to find out the name of a key combination and whether it will work on your system is to press the key while lf is running and read the name from the "unknown mapping" error.

Reply:
I have used your suggestion, thanks for this.